.github/workflows/lint-and-test.yml

```diff
@@ -1,11 +1,11 @@
-name: Lint and test
+name: Lint, test and build
```

Comment:
Should the markdown file name change?

Reply:
Maybe; I think building Eleventy is kind of a test.

Reply:
Renamed to check.